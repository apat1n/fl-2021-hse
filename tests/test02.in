(0, 1)
(q_1, q_2, q_3)
q_1
f(q_3)
[(q_1, q_2, (0)), (q_1, q_3, (1)), (q_2, q_2, (0, 1)), (q_3, q_3, (1)), (q_3, q_2, (0))]