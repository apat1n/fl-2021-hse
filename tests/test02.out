T_ALPHABET (0, 1) 1 0
T_STATES (q_1, q_2, q_3) 2 7
T_INITIAL q_1 3 23
T_FINAL_STATES f(q_3) 4 27
left-bracket [ 5 34
T_TRANSITION (q_1, q_2, (0)) 5 35
comma , 5 50
T_TRANSITION (q_1, q_3, (1)) 5 52
comma , 5 67
T_TRANSITION (q_2, q_2, (0, 1)) 5 69
comma , 5 87
T_TRANSITION (q_3, q_3, (1)) 5 89
comma , 5 104
T_TRANSITION (q_3, q_2, (0)) 5 106
right-bracket ] 5 121
(eof 5)
